Crust/Mapper/MappingOperator.swift

```diff
@@ -290,7 +290,7 @@ private func map<T, U: Mapping, V: Sequence>(
 }
 
 @discardableResult
-public func map<T, U: Mapping, C: MappingContext>(toCollection field: inout U.SequenceKind, using binding:(key: Binding<U>, context: C)) -> C where U.MappedObject == T, U.SequenceKind: RangeReplaceableCollection, U.SequenceKind.Iterator.Element == U.MappedObject, T: Equatable {
+public func map<T, U: Mapping, C: MappingContext, Coll: RangeReplaceableCollection>(toCollection field: inout Coll, using binding:(key: Binding<U>, context: C)) -> C where U.MappedObject == T, Coll.Iterator.Element == U.MappedObject, T: Equatable {
```

Comment:
`Collection` is an existing type constraint, I don't want to fight the compiler on that...

This could still use some improvement though. The rules I've been following on new code are:
1. Generics used in type constraints or where clauses get abbreviations matching their constraints/clause use e.g. `M: Mapping` or `RRC: RangeReplaceableCollection`. I prefer abbreviations here because it keeps the `where` clause a lot shorter, they can explode in verbosity pretty quickly, especially if you're referring to a generic's `.` typealias on typealias.
2. With no constraints/where clause fallback to the name of the argument.
3. Complexly constrained generics, no obvious choice, or otherwise fallback to T,U,V...